plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/visitors/FindDefinitionModelVisitor.java

```diff
@@ -391,6 +383,44 @@ public Object visitAssign(Assign node, int unpackPos) throws Exception {
         return super.visitAssign(node);
     }
 
+    /**
+     * @see org.python.pydev.parser.jython.ast.VisitorBase#visitIf(org.python.pydev.parser.jython.ast.If)
+     */
+    @Override
+    public Object visitIf(If node) throws Exception {
```

Comment:
Is it possible to do this in a `visitNamedExpr`?

i.e.: This seems to be missing the case where the walrus is in some other place (it could e something as `while a:= 10:)

See: https://pythoninformer.com/python-language/intermediate-python/walrus-operator/ for some examples (please extend test-cases to cover those).